### PR TITLE
Eng 10899 

### PR DESCRIPTION
**Pull request by luyangco**

### CODE REVIEW

File: src/frontend/org/voltdb/RealVoltDB.java [hunk 1]
```diff
@@ -1074,8 +1074,11 @@ public void initialize(Configuration config) {
                 VoltDB.crashLocalVoltDB(e.getMessage(), true, e);
             }
 
+            LicenseApi api = VoltDB.instance().getLicenseApi();
+            final boolean licensedToDR = api.isDrReplicationAllowed();
+
             // DR overflow directory
-            if (m_config.m_isEnterprise) {
+            if (m_config.m_isEnterprise && licensedToDR) {
```
Comment (nshi) [inline src/frontend/org/voltdb/RealVoltDB.java]: You can just call `m_licenseApi.isDrReplicationAllowed()` here.

**Comment by nshi** [comment]
Review finished.

File: src/frontend/org/voltdb/RealVoltDB.java (hunk 1 above)
Comment (luyangco) [inline src/frontend/org/voltdb/RealVoltDB.java]: I'll change that, thanks.

**Comment by luyangco** [comment]
Addressed the review comment, also rebased with master.

**Pull request merged**